google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -177,7 +471,11 @@ static DeleteRule fromPb(Rule rule) {
    * Delete rule class that sets a Time To Live for blobs in the bucket.
    *
    * @see <a href="https://cloud.google.com/storage/docs/lifecycle">Object Lifecycle Management</a>
+   *
+   * @deprecated Use a {@code LifecycleRule} with a {@code DeleteLifecycleAction} and use
+   * {@code LifecycleCondition.Builder.setAge} instead
```

Comment:
Can you add a small snippet in the documentation that demonstrates how to use this path? At least for this one.